units/getty@.service.m4

```diff
@@ -46,10 +46,6 @@ KillMode=process
 IgnoreSIGPIPE=no
 SendSIGHUP=yes
 
-# Unset locale for the console getty since the console has problems
-# displaying some internationalized messages.
-Environment=LANG= LANGUAGE= LC_CTYPE= LC_NUMERIC= LC_TIME= LC_COLLATE= LC_MONETARY= LC_MESSAGES= LC_PAPER= LC_NAME= LC_ADDRESS= LC_TELEPHONE= LC_MEASUREMENT= LC_IDENTIFICATION=
```

Comment:
At this moment, clearing locale environment would only fix translated `agetty` and `login` output. Once we get to the actual shell, it gets the locale environment again and everything is back. Either the shell reads it (as in Fedora/RHEL/etc.), or pam_env in PAM reads it (as in Debian/Ubuntu/etc).

So, if people set up locale to output characters which are not supported by the console, then they're OK with the output of the rest of the programs being broken, which is overwhelming majority of programs.

If people don't want their consoles outputting garbage, then they need to change LC_MESSAGES, LC_DATE, etc. to something that works (and maybe setup their environment differently for graphical sessions), or fix console output.

I think systemd shouldn't mess with locale environment for all possible locales just to fix `agetty` and `login` output for some that happen to be unsupported by console.

Getting to the point where systemd is the authoritative source of locale environment, and almost nothing needs to read or set the configuration itself, as mentioned in 1640944a, would require systemd stopping clearing those variables.